src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -31,12 +32,16 @@ public function testServicesAreOrderedAccordingToPriority()
             new Reference('n3'),
         );
 
-        $container = $this->getMockBuilder('Symfony\Component\DependencyInjection\ContainerBuilder')->setMethods(array('findTaggedServiceIds'))->getMock();
+        $container = $this->getMockBuilder('Symfony\Component\DependencyInjection\ContainerBuilder')->setMethods(array('findTaggedServiceIds', 'getDefinition'))->getMock();
```

Comment:
Do we really need to mock the `ContainerBuilder` class in the first place? IMO, most of the times we can simply use a "real" `ContainerBuilder` object when writing tests for compiler passes.